.cookiecutter/includes/README.md

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
This is the cookiecutter's "includes" feature: it appends the content of `.cookiecutter/includes/README.md` at the bottom of the real `README.md` after the standard templated README opener.

Reply:
Is it worth following the pattern of the other entry-points and assume there will be more in the future. By calling the "tail" entry-point after the whole file it precludes a nice layout like `README.md/tail` etc.

Reply:
Yeah I think that might be a nice pattern. By naming the include file itself `README.md` I've made it impossible to have an includes _directory_ named `README.md`. It'd be nice to be able to follow the same pattern for all files:

```
.cookiecutter/
  includes/
    README.md/
      head
      tail
      foo
      bar
```

Ofcourse `head` and `tail` may not make sense for every file but the folder-based layout can still apply.

This is actually a change to the cookiecutter rather than to this PR though

Reply:
As an aside I kind of hate having lots of separate files for this. I think what I'd like better is a single root `cookiecutter.toml` containing everything that's currently in `.cookiecutter/cookiecutter.json` and all the `.cookiecutter/include/` files. But that's not possible: `toml` isn't available to cookiecutters since it's not in the standard library yet, neither is YAML (also yuck), and JSON won't do because it doesn't support multiline strings,

Reply:
I'm gonna postpone this change until the cookiecutter itself gets a proper review